src/ui/public/doc_table/components/table_row/details.html

```diff
@@ -1,6 +1,8 @@
 <td colspan="{{ columns.length + 2 }}">
-  <a class="pull-right" ng-href="#/doc/{{indexPattern.id}}/{{row._index}}/{{row._type}}/?id={{row._id | uriescape}}">
-    <small>Link to /{{row._index}}/{{row._type}}/{{row._id | uriescape}}</small></i>
-  </a>
+  <div class="pull-right">
+    <a ng-href="{{docUrl}}">View</a>
+    <span style="display:inline-block; width:5px"><!--spacer--></span>
+    <a ng-click="copyTextToClipboard(docUrl)">Copy</a>
```

Comment:
I realized just the word "Copy" might be confusing to the user, they might think it's supposed to copy the json or something. How about changing this to "Copy URL"?